lib/common/http3client.c

```diff
@@ -504,21 +504,28 @@ static int handle_input_expect_headers(struct st_h2o_http3client_req_t *req, con
             return 0;
         }
         notify_response_error(req, *err_desc != NULL ? *err_desc : "qpack error");
+        if (*err_desc == NULL) {
+            *err_desc = "qpack error";
+        }
```

Comment:
```suggestion
        if (*err_desc == NULL)
            *err_desc = "qpack error";
        notify_response_error(req, *err_desc);
```
I think we can be less redundant?